tools/common/Target.cs

```diff
@@ -82,7 +82,7 @@ public PlatformLinkContext GetLinkContext ()
 
 		public void ExtractNativeLinkInfo (List<Exception> exceptions)
 		{
-			foreach (var a in Assemblies) {
+			foreach (var a in Assemblies.Where (x => !App.BindingProjectNoEmbedding.Contains (Path.GetFileName (x.FullPath)))) {
```

Comment:
I'm not sure about the option itself... extracting is slow (we want to avoid it)
However checking for embedding assemblies should be _fast_ (it's already loaded into memory anyway)
Is it worth the extra logic ?

IOW can we just extract when present (no check or additional option) and _benefit_ from the fact that newer/updated bindings don't embed (so we don't extract) ?

Reply:
side note: it might become apparent why when I read the next commits... but I'm reviewing (and commenting) as requested :)

Reply:
Executed - Once per build for each referenced assembly. 

I could rework the extract logic to not error, but it seemed easier to just skip it. It's a quick hashset lookup